doc/stomp_planner/stomp_planner_tutorial.rst

```diff
@@ -22,19 +22,20 @@ You should also have gone through the steps in `Visualization with MoveIt RViz P
 
 Prerequisites
 -------------
- 1. You must have the latest version of MoveIt for your ROS distribution installed. As STOMP is distributed independently you can safely install the packaged version of MoveIt however. STOMP then needs to be build from source, we will go through the steps for doing this below.
- 2. To use STOMP with your robot you must need to have a MoveIt configuration package for your robot ready. For example, if you have a Panda robot, it's probably called ``panda_moveit_config``. This is typically built using the `MoveIt Setup Assistant <../setup_assistant/setup_assistant_tutorial.html>`_.
+ 1. You must have the latest version of MoveIt for your ROS distribution installed. As STOMP is distributed independently you can safely install the packaged version of MoveIt. However, STOMP needs to be built from source. We will go through the steps for doing this below.
+ 2. To use STOMP with your robot you must have a MoveIt configuration package for your robot ready. For example, if you have a Panda robot, it's probably called ``panda_moveit_config``. This is typically built using the `MoveIt Setup Assistant <../setup_assistant/setup_assistant_tutorial.html>`_.
  3. You must also have built `ros-industrial/stomp_ros package <https://github.com/ros-industrial/stomp_ros>`_ and  `ros-industrial/stomp package <https://github.com/ros-industrial/stomp>`_ from source. This needs to be built from source since stomp_ros is not released as a debian yet.
 
 Installing STOMP from Source
 ------------------------------
-As you add and remove packages from your workspace you will need to clean your workspace and re-run the command to install new missing dependencies. Clean your workspace to remove references to the system wide installation of MoveIt: ::
+As you add and remove packages from your workspace you will need to clean your workspace with ``catkin clean`` and re-run the command to install new missing dependencies. Clean your workspace to remove references to the system wide installation of MoveIt: ::
 
-  cd ~/ws_moveit/src
+  cd ~/ws_moveit
   source /opt/ros/melodic/setup.bash
-  wstool init
-  wstool set stomp https://github.com/ros-industrial/stomp_ros.git --git
-  wstool update
+  wstool set -t src stomp_ros https://github.com/ros-industrial/stomp_ros.git --git
+  wstool update -t src stomp_ros
+  wstool merge -t src src/stomp_ros/dependencies.rosinstall
+  wstool update -t src stomp ros_industrial_cke_boilerplate
```

Comment:
```suggestion
  wstool merge -t src src/stomp_ros/dependencies.rosinstall
  wstool update -t src stomp ros_industrial_cmake_boilerplate
```
Do you really need to build those dependencies from source? Aren't they available as debians? 
If so, use `rosdep` to install them instead.

Reply:
I have fixed the typo. I think stomp needs to be built from source whereas ros_industrial_cmake_boilerplate could also be installed with apt. Thanks for the advice!